components/ws-daemon/pkg/content/initializer.go

```diff
@@ -192,7 +192,7 @@ func RunInitializer(ctx context.Context, destination string, initializer *csapi.
 		Destination: "/dst",
 		Source:      destination,
 		Type:        "bind",
-		Options:     []string{"bind", "rprivate"},
+		Options:     []string{"rbind", "rprivate"},
```

Comment:
To be honest, I don't know if we should change to `rbind` here.
But I was hoping this could address the issue #11852

@csweichel For this mount point, should we need the `rbind` instead of `bind`?

Reply:
I think `bind` here is more then enough, since we should not have any other binds inside `destination` param that used as source for the bind command.

Reply:
Agree with you @sagor999.

Close this PR first, and if issue #11852 still happens, we could revisit it to see what went wrong.